app/src/main/java/com/developer/abhinavraj/moviesearch/CustomListAdapter.java

```diff
@@ -0,0 +1,35 @@
+package com.developer.abhinavraj.moviesearch;
+
+import android.app.Activity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+public class CustomListAdapter extends ArrayAdapter {
+    private final Activity context;
+    private final String[] nameArray;
+    private final String[] infoArray;
+
+    public CustomListAdapter(Activity context,String[] nameArrayParam,String[] infoArrayParam ){
+        super(context, R.layout.listview_row,nameArrayParam);
+        this.context=context;
+        this.nameArray=nameArrayParam;
+        this.infoArray=infoArrayParam;
+
+    }
+    public View getView(int position, View view, ViewGroup parent)
+    {
+        LayoutInflater inflater=context.getLayoutInflater();
+        View rowView=inflater.inflate(R.layout.listview_row,null,true);
+
+        TextView nameTextField=(TextView)rowView.findViewById(R.id.nameTextViewID);
+        TextView infoTextField=(TextView)rowView.findViewById(R.id.infoTextViewID);
+
+        nameTextField.setText(nameArray[position]);
+        infoTextField.setText(infoArray[position]);
+
+        return rowView;
+    }
+}
```

Comment:
@ris3456 Please refactor your code, [StackOverFlowLink](https://stackoverflow.com/questions/16580171/code-formatting-shortcut-in-android-studio)

Reply:
And @ris3456 why have you made this adapter, this wasn't your task!!!

Reply:
@abhinavraj23 srry i was confused